src/bankAccount.js

```diff
@@ -1,7 +1,31 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+BankAccount.prototype = {
+  get amount() {
+    return Object.values(this.history)
+      .reduce((sum, current) => sum + current);
```

Comment:
старайся не проводить многоуровневые вычисления в return

```suggestion
  get amount() {
    const values = Object.values(this.history);
    
    return values.reduce((sum, current) => sum + current);
```